### ISSUE TITLE
Ghunt won't retrieve name

### ISSUE THREAD

**Issue by FatFarmerJoe** [mxrch/GHunt#515]
**Describe the bug**
Ghunt won't retrieve the name to a Google account. It was working until a little over a week ago.

**To Reproduce**
Simply run "ghunt email {email}" and it will give a bunch of information except a name.

**Expected behavior**
Retrieve the name to a google account/email.

**System (please complete the following information):**
 - OS (MacOS and Windows) I tried on 2 computers
 - Python version 3.11

**Comment by BruunAnd** [comment]
See this commit: https://github.com/mxrch/GHunt/commit/250747456ff4fa5d26b95d168a85715fec43e4d6

**Comment by FatFarmerJoe** [comment]
@BruunAnd Thanks! I did that and it still doesn't work. For some reason, it only works on some emails. I tested emails I have that previously worked, and only a couple actually returned the name. Something is wrong. Any thoughts? Thanks!

**Comment by FatFarmerJoe** [comment]
So I changed the "photo" option to "drive" in the peoplepa file and managed to get this information:
{'error': {'code': 403, 'message': 'Internal People API has not been used in project 341364422405 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/people-pa.googleapis.com/overview?project=341364422405 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.', 'status': 'PERMISSION_DENIED', 'details': [{'@type': 'type.googleapis.com/google.rpc.Help', 'links': [{'description': 'Google developers console API activation', 'url': 'https://console.developers.google.com/apis/api/people-pa.googleapis.com/overview?project=341364422405'}]}, {'@type': 'type.googleapis.com/google.rpc.ErrorInfo', 'reason': 'SERVICE_DISABLED', 'domain': 'googleapis.com', 'metadata': {'consumer': 'projects/341364422405', 'service': 'people-pa.googleapis.com'}}]}}

So something needs to be enabled? What can I do about this?

**Comment by BruunAnd** [comment]
My point with the commit reference was to show that the author of this repository says that names are patched.

It's definitely still possible, though - Epieos has a working solution. 

Regarding the Google API, that's an internal API key created by Google, so you unfortunately cannot enable the People API.

**Comment by FatFarmerJoe** [comment]
Oh, that makes sense. That's a bummer.

@Epieos If you don't mind me asking, how did you get it to work? Thanks!

**Comment by ersvrtgsrvbsrdb** [comment]
> Oh, that makes sense. That's a bummer.
> 
> @Epieos If you don't mind me asking, how did you get it to work? Thanks!

the name field is completely removed from epios, you can still send an email and maybe fetch their name like that

**Comment by BruunAnd** [comment]
> the name field is completely removed from epios, you can still send an email and maybe fetch their name like that

You're right. It was working for a few days after this repository got patched. Google is really on the hunt to close this extraction.

**Comment by mxrch** [comment]
For GHunt users / contributors : please don't try to bring back this feature, Google is actively monitoring the activity around this and adds new protections very quickly ; I think about the Epieos PR on GHunt that got blocked very quickly by Google.

As Google actively tries to stop the names extraction, consider this feature will not be back on GHunt. Goal of this tool is not to abuse Google services, just to automate what you can already do yourself. 

One way to display name would be to use the name on Google Maps again, but it'll appear only if there are reviews.

Cheers

**Comment by FatFarmerJoe** [comment]
MESSAGE FOR GOOGLE

Dear Google. Assuming you are reading this and not watching me through my webcam, please allow this feature as I use this for my job (legally) and this is to help me pay for college. Please focus your powers elsewhere where there is more harm being done on the internet. If not, that is fine. Thanks for letting us have the name feature up to this point.
